lib/model/resource.dart

```diff
@@ -7,47 +7,35 @@ extension ResourceExt on Resource {
     switch (this) {
       case Resource.CREW:
         return 'C';
-        break;
       case Resource.DOUBLOON:
         return 'D';
-        break;
       case Resource.FOOD:
         return 'F';
-        break;
       case Resource.INFAMY:
         return 'I';
-        break;
       case Resource.LANDLUBBER:
         return 'L';
-        break;
       case Resource.MAP:
         return 'M';
-        break;
       default:
         return '';
     }
   }
 
-  Color getColour() {
+  Color? getColour() {
```

Comment:
Hmm, again, this looks like it should be non-optional to me. Is it something about `Colours` that dart doesn't like? I guess maybe if you write `Colours.green[50234823048]` you could get null because that number doesn't exist? Lame, but again not important to sort for this PR I think!